ddtrace/span.py

```diff
@@ -493,6 +493,14 @@ def context(self):
             self._context = Context(trace_id=self.trace_id, span_id=self.span_id)
         return self._context
 
+    @property
+    def traceparent(self):
```

Comment:
The traceparent property should be consistent with the following spec:  https://www.w3.org/TR/trace-context/#traceparent-header

Reply:
From the spec: _This tag is a formatted composite value that includes the trace id, span id and a sampled flag. This is compliant with the w3c trace context spec. One example would be:_ `00-4bf92f3577b34da6a3ce929d0e0e4736-00f067aa0ba902b7-01`. 

So I think we'll need to convert the span_id and trace_id into hexadecimal values here.

Reply:
E.g: https://github.com/DataDog/dd-trace-py/blob/11dddbd05152c9493d7da7b9fd5316d38ea47696/ddtrace/propagation/http.py#L546